### DISCUSSION TITLE
Django-React URL Routing

### DISCUSSION THREAD

**Discussion by gabrielkarras** [gabrielkarras/SOEN341#70] [General]
We were debating whether React would handle URL routing via React Routing or would would that be done via Django's url.py

**Comment by gabrielkarras** [comment]
Up to now we it looks like we will use React Routing
# sources:
[StackOverflow-Can I Use React Router](https://stackoverflow.com/questions/52510439/can-i-use-react-router-in-django-project)
[StackOverflow-React Routing and Django URL Conflict](https://stackoverflow.com/questions/40826295/react-routing-and-django-url-conflict)

**Reply by minhtrannhat** [reply]
Yeah, I think we will use React Routing since it's better than Django regex based one. But there will be conflict between them so that's something to keep in mind.
